content/en/docs/gke/deploy/delete-cli.md

```diff
@@ -25,36 +20,19 @@ This guide assumes the following settings:
   export KF_DIR=<path to your Kubeflow application directory>
   ``` 
 
-* The `${CONFIG_FILE}` environment variable contains the path to your 
-  Kubeflow configuration file.
-
-  ```
-  export CONFIG_FILE=${KF_DIR}/{{% config-file-gcp-iap %}}
-  ```
-
-    Or:
+## Deleting your deployment
 
-  ```
-  export CONFIG_FILE=${KF_DIR}/{{% config-file-gcp-basic-auth %}}
-  ```
 
-For further background about the above settings, see the guide to
-[deploying Kubeflow with the CLI](/docs/gke/deploy/deploy-cli).
+1. To just delete the applications running in the Kubeflow namespace just delete that namespace
```

Comment:
Suggestion:

- Remove both occurrences of "just" to improve phrasing
- To avoid repeating "delete" twice in once sentence, replace the second one with "remove

```markdown
1. To delete the applications running in the Kubeflow namespace, delete that namespace
```

(Notice the comma separator.)

Reply:
Thanks